src/components/calcite-picker/calcite-picker-group/calcite-picker-group.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, Host, Prop, h } from "@stencil/core";
+
+@Component({
+  tag: "calcite-picker-group",
+  shadow: true
+})
+export class CalcitePickerGroup {
```

Comment:
It seems the suggestion is each component should have its own directory. It doesn't comment on whether to have all of those directories on a single level versus nesting them. https://stenciljs.com/docs/style-guide#file-structure
I'm ok with moving these all to a common level, but I find nesting second-class components into their first-class cousins lends a bit of a lay-up for understanding which is the main component and which are supporting components.

Reply:
But in their example there aren't any nested components.

```
├── card
│   ├── card.ios.css
│   ├── card.md.css
│   ├── card.css
│   ├── card.tsx
│   └── test
│       └── basic
│           ├── e2e.js
│           └── index.html
├── card-content
│   ├── card-content.ios.css
│   ├── card-content.md.css
│   ├── card-content.css
│   └── card-content.tsx
├── card-title
│   ├── card-title.ios.css
│   ├── card-title.md.css
│   ├── card-title.css
```

I think it makes sense to keep them all at root since they are all `HTMLElement`s.